client/src/pages/MainTable.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
+
+const MainTable = () => {
+  console.log("Page Uploaded");
+  const [clientData, SetClientData] = useState (null);
+  const [error, setError] = useState(null);
+  const [selectedClientId, setSelectedClientId] = useState (null);
+
+  useEffect(() => {
+    fetch("/api/clients")
+      .then((response) => {
+        if (!response.ok) {
+          throw Error("We could not fetch the data for that resource");
+          }
+          return response.json();
+        })
+      .then((data) => {
+        console.log(data,"After");
+        SetClientData(data);
+        console.log(clientData, "Data from useState");
+        setError(null);
+      })
+      .catch((err) => {
+              console.log("error message: ", err.message);
+              setError(err.message);
+      });
+  }, []);
+
+  console.log(clientData, "outside of fetch");
+
+if (clientData === null){
+  return <h1>Loading...</h1>;
+}
+  return (
+    <div className="container">
+      <table className="table table-sm table-striped table-hover">
+        {/* <caption>List of users</caption> */}
+        <thead>
+          <tr>
+            <th scope="col">Client Id</th>
+            <th scope="col">Added</th>
+            <th scope="col">Name</th>
+            <th scope="col">Phone</th>
+            <th scope="col">Bikes needed</th>
+            <th scope="col">Booking status</th>
+            <th scope="col">Residency status</th>
+            <th scope="col">Country of origin</th>
+            <th scope="col">Time in Scotland</th>
+            <th scope="col">Language spoken</th>
```

Comment:
It looks like there's a `Language spoken` column here but there's no `td` in the body for language spoken — can we fix this?

Reply:
Already included